HACKING.txt

```diff
@@ -57,52 +57,28 @@ Coding Style
 Running Tests
 -------------
 
-- The `tox.ini` uses `nose` and `coverage`. As such `tox` may be used
+- The `tox.ini` uses `pytest` and `coverage`. As such `tox` may be used
   to run groups of tests or only a specific version of Python. For example, the
-  following command will run tests on Python 3.7 only without coverage:
+  following command will run tests on Python 3.8 only with coverage:
 
-    $ tox -e py37
+    $ tox -e py38
 
-  This command will run tests on the latest version of Python 3 with coverage.
-
-    $ tox -e py36-cover,coverage
-
-- To run individual tests (i.e., during development), you can use `nosetests`
+- To run individual tests (i.e., during development), you can use `pytest`
   syntax as follows, where `$VENV` is an environment variable set to the path
   to your virtual environment:
 
     # run a single test
-    $ $VENV/bin/nosetests tests.test_module:ClassName.test_mytestname
+    $ tox -e py38 -- --no-cov tests/test_httpexceptions.py::TestHTTPMethodNotAllowed::test_it_with_default_body_tmpl
 
     # run all tests in a class
-    $ $VENV/bin/nosetests tests.test_module:ClassName
-
-  Optionally you can install a nose plugin `nose-selecttests` to run specific
-  tests.
-  https://pypi.org/project/nose-selecttests/
-  For example, use a regular expression with the `-t` parameter to run tests.
-
-    # run a single test
-    $ $VENV/bin/nosetests -t test_mytestname
-
-- The tests can also be run using `pytest`.
-  https://docs.pytest.org/en/latest/
-  This is intended as a convenience for people who prefer `pytest`. Run the
-  tests like so:
-
-    $ $VENV/bin/pip install pytest
-    $ $VENV/bin/pytest --strict pyramid/ tests/
-
-  To run individual tests (i.e., during development), see "pytest usage -
-  Specifying tests / selecting tests":
-  https://docs.pytest.org/en/latest/usage.html#specifying-tests-selecting-tests
+    $ tox -e py38 -- --no-cov tests/test_httpexceptions.py::TestHTTPMethodNotAllowed
```

Comment:
I like the link to [pytest usage - Specifying tests / selecting tests](https://docs.pytest.org/en/latest/usage.html#specifying-tests-selecting-tests). I think it might help with on-boarding. Maybe have one abstract example and the link for additional invocations?

Reply:
I've added that link back in.